app/models/ability.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class Ability
+  include CanCan::Ability
+
+  def initialize(user)
+    if user.present?
```

Comment:
```suggestion
    return unless user.present?
```

Suggest using Rubocop :)

Reply:
@Yehezkiel01

Reply:
Btw, karena cm butuh minor change, I will add this change di next PR skalian fix bbrp minor changes lain ya buat code styling.